src/test/java/org/javarosa/core/model/instance/geojson/GeoJsonExternalInstanceTest.java

```diff
@@ -124,4 +124,9 @@ public void parse_throwsException_whenGeometryNotPoint() {
             // expected
         }
     }
+
+    @Test
+    public void parse_ignoresUnknownToplevelProperties() throws IOException {
+        TreeElement featureCollection = GeoJsonExternalInstance.parse("id", r("feature-collection-extra-toplevel.geojson").toString());
```

Comment:
Did this crash before or is there a missing assertion?

Reply:
Yes and yes. 😫 Fixed and force pushed the last commit. Please review that one again, it was really a disaster.